### PR TITLE
Add search view to header

### PR DESCRIPTION
**Pull request by jose-correia**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:
- Before: https://main--moleculardevices--hlxsites.hlx.page/
- After: https://header-search-view--moleculardevices--hlxsites.hlx.page/

### CODE REVIEW

File: blocks/header/header.css [hunk 1]
```diff
@@ -865,6 +870,33 @@ header .actionable-card-submenu > div:nth-child(2) > div:nth-child(2) img {
...
+  header .searchlink > div {
+    display: none;
+  }
+
+  header .header-search > div {
+    margin-top: 31px;
+    left: 0;
+    right: 0;
+    border: 0;
+    padding: 50px 50px 60px 100px;
```
Comment (paolomoz) [inline blocks/header/header.css:882]:
```suggestion
    padding: 47px 50px 60px 100px;
```

File: blocks/header/header.css [hunk 2]
```diff
@@ -899,6 +931,65 @@ header .actionable-card-submenu > div:nth-child(2) > div:nth-child(2) img {
...
+  header .menu-nav-search-bar {
+    padding: 0 20px;
+  }
+
+  header .menu-nav-search-bar h3 {
+    text-align: left;
+    font-size: 20px;
+    font-family: 'Proxima Nova Light', sans-serif;
+    color: white;
+    margin-bottom: 15px;
```
Comment (paolomoz) [inline blocks/header/header.css:956]:
```suggestion
    margin-bottom: 11px;
```

**Review by paolomoz — COMMENTED** [review]
added a couple of suggestions to fix vertical positioning

**Comment by paolomoz** [comment]
@jose-correia looking at horizontal positioning: 
.nav-tabs .searchlink > div should have width 1170px for a large screen (I think it has 1140 now) with padding: 50px 100px
and .robot-image-with-speech and .menu-nav-search-bar should be 50% each, both with padding: 0 15px;

File: blocks/header/header.js [hunk 3]
```diff
@@ -24,10 +32,45 @@ function buildRequestQuote() {
...
+          h3(
+            'Search',
+          ),
+          div(
+            { class: 'search-form-group' },
+            form(
+              { action: '/search-results', method: 'GET' },
+              input(
+                {
+                  id: 'search_keyword_search1', class: 'form-control', type: 'text', placeholder: 'moleculardevices.com', name: 'search',
```
Comment (mhaack) [inline blocks/header/header.js:57]: I think the query parameter aka name of the field for the search page should be `q=`

Comment (jose-correia) [reply blocks/header/header.js]: Fixed this by adding additional JS. According to Moldev team, we need to maintain this format

**Review by mhaack — APPROVED** [review]

File: styles/styles.css [hunk 4]
```diff
@@ -81,6 +81,7 @@
   --header-mega-menu-product-cards-background-color: #161e2d;
   --header-mega-menu-green: #6eb43f;
   --header-actionable-submenu-text-color: #e6e6e6;
+  --input-grey-text: #555;
```
[the remark this answers is not in the document]
Comment (jose-correia) [reply styles/styles.css:84]: yeah, you're right

**Pull request merged**